os/fs/smartfs/README.txt

```diff
@@ -1,90 +1,179 @@
-SMARTFS README
+jSmartFS README
 ^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
 Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
 
+The same has now been upgraded to jSmartFs with the addition of 2 key features -
+Filesystem Consistency and Two-Tier File Data Integrity.
+
 Contents:
 
-  Features
-  General operation
-  SMARTFS organization
-  Headers
-  Multiple mount points
-  SMARTFS Limitations
-  ioctls
-  Things to Do
+  > Features
+  > General operation
+  > SMARTFS organization
+  > Headers
+  > Multiple mount points
+  > SMARTFS Limitations
+  > ioctls
+  > Things to Do
 
 Features
 ========
 
   This implementation is a full-feature file system from the perspective of
-  file and directory access.  The SMART File System was designed specifically
+  file and directory access. The SMART File System was designed specifically
   for small SPI based FLASH parts (1-8 Mbyte for example), though this is not
-  a limitation.  It can certainly be used for any size FLASH and can work with
-  any MTD device by binding it with the SMART MTD layer and has been tested with
-  devices as large as 128MByte (using a 2048 byte sector size with 65534 sectors).
+  a limitation.  It can certainly be used for flash memories of any size and has
+  been tested with devices as large as 128MiB (using a 2048 byte sector size
+  with 65534 sectors).
   The FS includes support for:
+    - Support for multiple mount points on a single volume / partition.
```

Comment:
```suggestion
    - Multiple mount points on a single volume / partition.
```
"Support for" is a repetition from the previous line.

Reply:
Updated, thankyou